helmfile.d/helmfile-07.init.yaml

```diff
@@ -18,3 +18,9 @@ releases:
     labels:
       pkg: cert-manager
     <<: *raw
+  - name: minio-artifacts
+    installed: {{ $a | get "minio.enabled" }}
```

Comment:
since there are defaults you can simply do
```
$a.minio.enabled
```

Reply:
fixed